paper/paper.md

```diff
@@ -0,0 +1,88 @@
+<!--
+SPDX-FileCopyrightText: ASSUME Developers
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+---
+title: 'ASSUME - Agent-based Simulation for Studying and Understanding Market Evolution'
+tags:
+  - Python
+  - agent based simulation
+  - energy market
+  - reinforcement learning
+  - market simulation
+  - simulation
+authors:
+  - name: Florian Maurer
+    orcid: 0000-0001-8345-3889
+    corresponding: true
+    affiliation: 1
+    "affiliation": 1
+  - name: Nick Harder
+    orcid: 0000-0003-1897-3671
+    "affiliation": 2
+  - name: Kim K. Miskiw
+    orcid: 0009-0009-1389-4844
+    affiliation: 3
+  - name: Manish Khanra
+    orcid: 0000-0002-3347-9922
+    affiliation: 4
+
+affiliations:
+ - name: University of Applied Sciences Aachen, Germany
+   index: 1
+ - name: Inatech Universität Freiburg, Germany
+   index: 2
+ - name: Karlsruhe Institute of Technology, Germany
+   index: 3
+ - name: Fraunhofer Institute for Software and Systems Engineering, Germany
+   index: 4
+date: 13 November 2024
+bibliography: paper.bib
+---
+
+<!-- pandoc -s paper.md -o paper.pdf --bibliography paper.bib --csl=apa.csl --filter pandoc-citeproc --pdf-engine=xelatex-->
+
+# Summary
+
+**ASSUME** is an open-source toolbox for agent-based simulations of various energy markets, with a primary focus on European electricity markets. Developed as an open-source model, its objectives are to ensure usability and customizability for a wide range of users and use cases in the energy system modeling community.
+
+A unique feature of the `ASSUME` toolbox is its integration of **Deep Reinforcement Learning** methods into the behavioral strategies of market agents. The model offers various predefined agent representations for both the demand and generation sides, which can be used as plug-and-play modules, simplifying the usage of reinforcement learning strategies. This setup enables research of new market designs and evolving dynamics in energy markets.
+
+# Statement of need
+
+While other open-source agent-based models have been developed for studying energy markets, such as PowerACE [@bublitzAgentbasedSimulationGerman2014] and AMIRIS [@schimeczekAMIRISAgentbasedMarket2023], the possible integration of reinforcement learning methods into the behavioral strategies of market agents is currently unique to `ASSUME`.
```

Comment:
here maybe add that they typically rely on rule-based bidding strategy representation, which limits the ability to represent future markets or alternative markets designs, as in reality agents would adapt to the new reality.